src/containers/checkout/checkout-form/checkout-form.js

```diff
@@ -145,12 +137,7 @@ const CheckoutForm = ({ cartItems, promoCode, certificate, handleCashPayment })
 
   useEffect(() => {
     if (userData) {
-      setInitialValues(
-        updateInitialValues(
-          { ...userData, phoneNumber: userData.phoneNumber.slice(3) },
-          deliveryType
-        )
-      );
+      setInitialValues(updateInitialValues({ ...userData }, deliveryType));
```

Comment:
can you use just "updateInitialValues(userData, deliveryType)"?

Reply:
no, it's just a helper function that returns an object, and we need to save it